crates/grafbase-sdk/Cargo.toml

```diff
@@ -37,16 +42,15 @@ jq-selection = [
 ]
 
 [dependencies]
-anyhow.workspace = true
+document-features = { version = "0.2" }
```

Comment:
I think you're missing an `optional = true` here, since you declared that as a feature above in the package.metadata.docsrs section.

Also TIL about this crate, neat!

Reply:
ah right, let me retry it, it didn't really worked out like I expected

Reply:
Well I just can get it to work, giving up